src/containers/Announcements.js

```diff
@@ -12,6 +12,7 @@ export default () => {
       .doc(DB_HACKATHON)
       .collection('Announcements')
       .orderBy('timestamp', 'desc')
+      .limit(6)
       .onSnapshot(querySnapshot => {
```

Comment:
can we reuse the data fetched from `src/App.js:L117`?

Reply:
let's get someone with iOS to test the preview pr to make sure my changes didn't break anything before we merge. @acchiang?

Reply:
just got carmen to test on iOS, looks good